### DISCUSSION TITLE
Expanding the docs on Expo

### DISCUSSION THREAD

**Discussion by valerius21** [tamagui/tamagui#485] [General]
Hi devs!

I'd love to see some explaination on how the linked repo's are to be understood, because I'm trying to integrate tamagui into an older project and don't see applying solito.dev at the moment. 

Since I don't think, the complete monopreo is a necessity and a "please look at this massive example" makes it quite difficult to understand the integration process of tamagui into an existing Expo project, I'd love some more details on how this is to be archieved :)

What are your thougts? 

Cheers!

**Comment by izakfilmalter** [comment]
Hey @valerius21, I am in the process of putting Tamagui into a brownfield repo. I downloaded the solito example and copied the `packages/ui` directory into my app. I then manually diffed the files in `apps/expo` with my app to get it working.

**Comment by Mohamed-kassim** [comment]
@valerius21 there is also
[ivopr/tamagui-expo](https://github.com/ivopr/tamagui-expo)
Tamagui Native only expo template without mono repo
But In general all you need is just

install tamagui directly
Add Tamagui config with types
Add TAMAGUI_TARGET env var to native
make sure you add transform-inline-environment-variables in babel
if you need to enable compiler, you need to enable it's babel configuration and required package
Also make sure to check react Dom version, if I remember latest version results in some hook error

But I agree about docs.

**Reply by astriskit** [reply]
@Mohamed-kassim  Could you please comment on using tamagui in expo without the web-support? 

The motivation is that just adding the whole tamagui ui-kit(`yarn expo install tamagui`) brings in `@tamagui/web` and yarn/npm complains about missing peer dependencies of "react-native-web" and "react-dom" which may not be necessary for someone targeting just the native using expo.

Thanks.
